src/avram/query_builder.cr

```diff
@@ -298,17 +298,35 @@ class Avram::QueryBuilder
     where(where_clause)
   end
 
+  def or(&block : Avram::QueryBuilder -> Avram::QueryBuilder)
+    if @wheres.empty?
+      raise Avram::InvalidQueryError.new("Cannot call `or` before calling a `where`")
+    end
+
+    @wheres.last.conjunction = Avram::Where::Conjunction::Or
+
+    block.call(self)
```

Comment:
@matthewmcgarvey maybe you can see what's up here, but I'm not sure what I'm missing? 

If you look at the failing spec, you'll see that it's expecting `users.age` to be there 3 times, but it's only there once. This is the line where it calls the block which should hook in to the existing QueryBuilding to append `age` the second and third time. This worked prior to merging in master, but now it doesn't 🤔

Reply:
@jwoertink I guess this PR was made after the update to immutable QueryBuilder/Queryable. You need to return this line, not the one below it.  The same in the other or method.

Reply:
Ah! You're right, I forgot about that. I'll give that a shot. Thanks!